bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/AjaxNumericBootstrapSlider.java

```diff
@@ -0,0 +1,25 @@
+package de.agilecoders.wicket.extensions.slider;
+
+import de.agilecoders.wicket.extensions.slider.util.*;
+import org.apache.wicket.model.IModel;
+
+/**
+ * AjaxNumericBootstrapSlider
+ */
+public class AjaxNumericBootstrapSlider<T extends Number> extends AjaxBootstrapSlider<INumericValue<T>, T> {
+
+    public AjaxNumericBootstrapSlider(String id, IModel<T> model) {
+        super(id, new NumericModel(model), (Class<INumericValue<T>>)getSliderClass(model.getObject().getClass()));
```

Comment:
`(Class<INumericValue<T>>)getSliderClass(model.getObject().getClass())` - why this is needed?
Reading the model object so soon is usually bad practice.